public/content/docs/mels-hack/index.en.md

```diff
@@ -0,0 +1,173 @@
+---
```

Comment:
General: This PR presents a bug in the Responsive View. Paragraphs are not wrapping correctly, and the text is is cut. Please check this.

Reply:
The broken display is due to a `display:table` style under a radix scroll area. When I remove this attribute, the text flows well in mobile mode.

![image](https://user-images.githubusercontent.com/180638/198151756-0f6b186a-bfda-4872-85f7-b915857eaddc.png)